debezium-core/src/main/java/io/debezium/transforms/outbox/SchemaBuilderUtil.java

```diff
@@ -148,4 +148,8 @@ private static Schema buildDocumentUnionSchema(ArrayNode array) {
         }
         return builder.build();
     }
+
+    private static boolean hasNotField(SchemaBuilder builder, String fieldName) {
```

Comment:
Can we name this `hasField` and invoke it above with `!hasField(...)`? I.e. let's rely on the canonical negation operator rather than negating within the name itself. It will become bothersome when looking for a field that's present ( `!hasNotField(...)`).

Reply:
I'm just a guest here, so I won't argue and apply the suggestion (but personally if `hasField` was needed I would just refactor `hasNotField` to add `hasField` method and refactor `hasNotField` to negate the result of `hasField` 🤷‍♂️ )

Reply:
btw any reasons why expanding the JSON content is that complicated? I was expecting to find a one-liner, sth like: `org.apache.commons.text.StringEscapeUtils.unescapeJson`

Reply:
> btw any reasons why expanding the JSON content is that complicated? I was expecting to find a one-liner, sth like: org.apache.commons.text.StringEscapeUtils.unescapeJson

Oh, sure, I'd love the one line if you can provide one (without adding a new dependency just for that) ;)